script.js

```diff
@@ -0,0 +1,641 @@
+if (!localStorage.currenLang) localStorage.currenLang = 'en';
+let altPressed = false;
+let capsLockEnabled = false;
+let shiftPressed = false;
+const lang = localStorage.currenLang;
+let secondLang;
+
+lang === 'en' ? secondLang = 'ru' : secondLang = 'en';
+document.body.insertAdjacentHTML('afterbegin', '<div class="wrapper" id="wrapper"><textarea  class="area" id="result" rows="5" cols="30"></textarea><div class="keyboard" id="keyboard"></div>');
+for (let i = 0; i < config.length; i++) {
+  const divLine = document.createElement('div');
+  divLine.classList.add('line');
+  for (let j = 0; j < config[i].length; j++) {
+    const divKey = document.createElement('div');
+    divKey.classList.add('key');
+    divKey.classList.add(`key-${config[i][j].key}`);
+    divKey.insertAdjacentHTML('afterbegin', `<div class="${config[i][j].key} show">
+        <span class="${lang} lower active">${config[i][j][`${lang} lower`]}</span>
+        <span class="${lang} upper disabled">${config[i][j][`${lang} upper`]}</span></div>
+        <div class="${config[i][j].key} disabled">
+        <span class="${secondLang} lower active">${config[i][j][`${secondLang} lower`]}</span>
+        <span class="${secondLang} upper disabled">${config[i][j][`${secondLang} upper`]}</span></div>`);
+    divLine.append(divKey);
+  }
+  keyboard.append(divLine);
+}
+wrapper.append(keyboard);
+
+function changeLang() {
+  const currentLang = keyboard.querySelectorAll('.key');
+
+  for (let i = 0; i < currentLang.length; i++) {
+    const firstChildLang = currentLang[i].children[0].classList[1];
+    const secondChildLanf = currentLang[i].children[1].classList[1];
+    currentLang[i].children[0].classList.remove(firstChildLang);
+    currentLang[i].children[0].classList.add(secondChildLanf);
+    currentLang[i].children[1].classList.remove(secondChildLanf);
+    currentLang[i].children[1].classList.add(firstChildLang);
+  }
+  let localLang = keyboard.querySelector('.show');
+  localLang = localLang.querySelector('span');
+  localStorage.currenLang = localLang.classList[0];
+}
+
+function keyLightOn(active) {
+  active.closest('.key').classList.add('clicked');
+}
+
+function keyLightOff(active) {
+  active.closest('.key').classList.remove('clicked');
+}
+
+function capsLetters(active) {
+  if (capsLockEnabled) {
+    keyLightOff(active);
+    capsLockEnabled = false;
+  } else {
+    keyLightOn(active);
+    capsLockEnabled = true;
+  }
+
+
+  const bigLetter = keyboard.querySelectorAll('.show');
+  for (let i = 0; i < bigLetter.length; i++) {
+    const firstChildLetter = bigLetter[i].children[0].classList[2];
+    const secondChildLetter = bigLetter[i].children[1].classList[2];
+    bigLetter[i].children[0].classList.add(secondChildLetter);
+    bigLetter[i].children[0].classList.remove(firstChildLetter);
+    bigLetter[i].children[1].classList.add(firstChildLetter);
+    bigLetter[i].children[1].classList.remove(secondChildLetter);
+  }
```

Comment:
as you have same logic above (even hardcoded) you could reuse creating separate function